tools/unix/generate_localizations.sh

```diff
@@ -48,8 +62,10 @@ STRINGS_UTILS="$OMIM_PATH/tools/python/strings_utils.py"
 "$STRINGS_UTILS" --types-strings --validate --output
 
 # Check for unused strings.
-CLEAN_STRINGS="$OMIM_PATH/tools/python/clean_strings_txt.py"
-"$CLEAN_STRINGS" --validate
+if [ -z $OPT_IGNORE_UNUSED ]; then
+  CLEAN_STRINGS="$OMIM_PATH/tools/python/clean_strings_txt.py"
+  "$CLEAN_STRINGS" --validate
```

Comment:
it doesn't auto remove unused strings, right? just checks they're present?

Reply:
yup, on fail there's advice to run a separate python script to clean strings